[RM Ch03] Introduction (#1780)
cntt-n/CNTT: doc/ref_model/chapters/chapter03.md

* [3.4 Physical Resources](#3.4)
* [3.5 Network](#3.5)

> There is the necessity to clearly define which kind of infrastructure resources a shared cloud infrastructure (network function virtualisation infrastructure, NFVI) will provide for hosting workloads including virtual network functions (VNFs) and/or cloud-native network functions (CNF), so that the requirements of the workloads match the capabilities of the cloud infrastructure.

Reviewer: Not sure what is being conveyed here including the beginning.  Are we defining the infrastructure resources that the cloud infrastructure will provide to the workloads? If yes, then the workloads must utilise the capabilities offered -- how would that lead to ("so that") "requirements of the workloads match the capabilities."  The following is a suggestion but still not satisfactory.
```suggestion
It is necessary to clearly define the infrastructure resources and their capabilities a shared cloud infrastructure (network function virtualisation infrastructure, NFVI) will provide for hosting workloads including virtual network functions (VNFs) and/or cloud-native network functions (CNFs).
```

Author: @pgoyal01, I agree.  This was a left-over form the original text. I will commit the suggestion and I will suggest you approving this PR.  We can polish up some of this later on.

The lack of a common understanding of which resources and corresponding capabilities a suitable cloud infrastructure should provide may lead to several issues which could negatively impact the time and the cost for on-boarding and maintaining these solutions on top of a virtualised infrastructure. For Example:

> - Supporting any kind of workload specific requirements (e.g. regarding network acceleration or API access) might result in having to establish different silo of cloud infrastructures for each workload type.

Reviewer: Not sure what this paragraph is trying to convey. Also, the paragraph that follows then reverts back to what is stated in the preceding paragraph.

Author: I have simplified the text removing the examples.  In this case, the less the better.

- Synchronising the release cycles of a large set of different technologies will sooner or later lead to situations in which required upgrades cannot be applied easily due to incompatibilities.

> The abstraction model presented in this chapter specifies a common set of virtual infrastructure resources which a cloud infrastructure will need to provide to be able to host most of the typical VNF/CNF workloads required by the operator community.

Reviewer: ```suggestion
The abstraction model presented in this Reference Model (RM) specifies a common set of virtual infrastructure resources that a cloud infrastructure will need to provide to be able to host most of the typical VNF/CNF telco workloads.
```

> Although a couple of explicit and implicit abstraction models (e.g. in the context of ETSI NFV) are already available, they fall short when addressing the following design principles:

Reviewer: DO we need a reference to these models that fall short?

Author: @pgoyal01, I have removed this sentence as being to contentious and unnecessary.  I do not think that we need to justify our efforts any more...

> - **Scope:** the model should describe the most relevant virtualised infrastructure resources (incl. acceleration technologies) a cloud infrastructure needs to provide for hosting Telco workloads

Reviewer: ```suggestion
- **Scope:** the model should describe the most relevant virtualised infrastructure resources (incl. acceleration technologies) a cloud infrastructure needs to host Telco workloads
```

- **Separation of Concern:** the model should support a clear distinction between the responsibilities related to maintaining the network function virtualisation infrastructure and the responsibilities related to managing the various VNF workloads
- **Simplicity:** the amount of different types of resources (including their attributes and relationships amongst one another) should be kept to a minimum to reduce the configuration spectrum which needs to be considered
- **Declarative**: the model should allow for the description of the intended state and configuration of the cloud infrastructure resources for automated life cycle management

> - **Explicit:** the model needs to be rich enough to allow for a direct mapping towards the APIs of the cloud infrastructure for the instantiation of virtual infrastructure elements without requiring any additional parameters

Reviewer: Not sure what it means to state, "allow for a direct mapping towards the APIs" and what parameters exist that the statement, "without requiring any additional parameters", is needed?

Author: @pgoyal01, I have changed it to 
- **Explicit:** the model needs to be rich enough to allow for the instantiation and the on-going operation of the cloud infrastructure

I guess that was the intent of this principle if first place.  Any further thoughts?

- **Lifecycle:** the model must distinguish between resources which have independent lifecycles but should group together those resources which share a common lifecycle

> - **Aligned:** the model should clearly highlight the dependencies between the elements to allow for a well-defined and simplified synchronisation of independent automation tasks.

Reviewer: which "elements"?

Author: @pgoyal01, I have changed to "its components", please check my commit.
